### PR TITLE
Feature/extend policy

### PR DESCRIPTION
**Pull request by yusank**

### CODE REVIEW

File: deploy/deploy.yaml [hunk 1]
```diff
@@ -39,10 +39,11 @@ spec:
             - ./webhook
           args:
             - --cert-dir=/certs
-            - --v=2
+            - --v=4
             - --logtostderr=false
+            - --alsologtostderr=true
             - --log_dir=./log
-          image: registry.cn-hangzhou.aliyuncs.com/k-cloud-labs/kinitiras-webhook:v0.1.1
+          image: docker.io/yusank/kinitiras-webhook:v0.1.2-01
```
Comment (likakuli) [inline deploy/deploy.yaml]: use origin repo

File: deploy/deploy.yaml [hunk 2]
```diff
@@ -72,6 +73,9 @@ spec:
             - mountPath: /kinitiras/log
               name: log
               subPathExpr: $(POD_NAME)
+            - mountPath: /etc/templates
```
Comment (likakuli) [inline deploy/deploy.yaml]: Is this needed?

File: deploy/deploy.yaml [hunk 3]
```diff
@@ -80,4 +84,7 @@ spec:
           hostPath:
             path: /var/log/kinitiras
             type: DirectoryOrCreate
-
+        - name: templates
```
Comment (likakuli) [inline deploy/deploy.yaml]: same as above

File: deploy/deploy.yaml (hunk 1 above)
Comment (yusank) [reply deploy/deploy.yaml]: ok I'll change it now.

File: go.mod [hunk 4]
```diff
@@ -1,14 +1,16 @@
 module github.com/k-cloud-labs/kinitiras
 
-go 1.17
+go 1.18
 
 require (
+	github.com/evanphx/json-patch v4.12.0+incompatible
 	github.com/go-logr/logr v1.2.3
-	github.com/k-cloud-labs/pkg v0.3.0
+	github.com/k-cloud-labs/pkg v0.3.1-0.20221107074337-9d43a07930c7
```
Comment (likakuli) [inline go.mod]: update it when pkg released new version

File: deploy/deploy.yaml (hunk 2 above)
Comment (yusank) [reply deploy/deploy.yaml]: Not any more, add this for more convenient for debug and test, I'll remove it.

File: go.mod (hunk 4 above)
Comment (yusank) [reply go.mod]: yes, waiting for new valid version of `pkg`

**Review by likakuli — APPROVED** [review]

**Pull request merged**